README.md

```diff
@@ -39,3 +39,15 @@ The build contains all wheels required by branches:
 .\Build-Wheels.ps1 -Branch "master" -Arch "-arm64" -CompileWheels @("greenlet", "gevent<2.0,>=1.2.2", "cryptography", "windows-curses", "python-pkcs11") -Python python3.9
 .\Test-Wheels.ps1 -Branch "master"
 ```
+
+## Requirements lists
+These lists are .txt files for requirements which should be added or excluded from the main requirements list which is automatically assembled.
+
+### include_list.txt
+File for additional Python packages to the main requirements list.
```

Comment:
Just to be clear, this additions should be added in a separate build in order to not restrict the "main list". The restrictions are put there by the exclude list.